File: The Core/At The Crossroads/12. Is Infinite Process/isInfiniteProcess.lua
function isInfiniteProcess(a, b)
    
    if a > b then
        return true
    end
    
    if a == b then 
        return false
    end
    
    if a < b then
        if (b-a) % 2 == 0 then
            return false
        else
            return true
        end
    end       
end

File: The Core/At The Crossroads/12. Is Infinite Process/isInfiniteProcess.py
def isInfiniteProcess(a, b):

    if a > b:
        return True
    
    if a == b:
        return False
    
    if a < b:
        if(b-a) % 2 == 0:
            return False
        else:
            return True
        
    return False
